docs/tutorial/Split_Learning_for_bank_marketing.ipynb

```diff
@@ -1698,6 +1698,14 @@
     "single_model.evaluate(test_data, test_label, batch_size=128)"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Referring to the above visualization code, the training process of a local model can also be visualized."
```

Comment:
No CN for `Referring to the above visualization` in po file ?

Reply:
I run
```
(sf) limingbo@DESKTOP-mingbo:/mnt/d/WorkShop/secretflow/fl/secretflow/docs$ sh update_po.sh
: not foundh: 2:
Running Sphinx v5.3.0
loading pickled environment... done
[autosummary] generating autosummary for: api/index.rst, awesome-pets/awesome-pets.md, awesome-pets/papers/applications/aml/attack_defense.md, awesome-pets/papers/applications/aml/poison.md, awesome-pets/papers/applications/multimedia/attack.md, awesome-pets/papers/applications/multimedia/defense.md, awesome-pets/papers/applications/multimedia/summary.md, awesome-pets/papers/applications/ppml/fl/Effectiveness.md, awesome-pets/papers/applications/ppml/fl/Efficiency.md, awesome-pets/papers/applications/ppml/fl/application.md, ..., tutorial/lr_with_spu.ipynb, tutorial/mix_lr.ipynb, tutorial/nn_with_spu.ipynb, tutorial/numpy_with_spu.ipynb, tutorial/practical_case_walkthrough_using_sf_with_spu.ipynb, tutorial/risk_control_scenario.ipynb, tutorial/split_learning_gnn.ipynb, tutorial/spu_basics.ipynb, tutorial/teeu.md, tutorial/teeu_xgboost.md
```
However，
```
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/practical_case_walkthrough_using_sf_with_spu.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/PSI_On_SPU.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/risk_control_scenario.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/SecureBoost.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/Split_Learning_for_bank_marketing.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/split_learning_gnn.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/spu_basics.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/teeu.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/teeu_xgboost.po
: not foundh: 4:
po files has been updated. Please update po files in locales folder.
(sf) limingbo@DESKTOP-mingbo:/mnt/d/WorkShop/secretflow/fl/secretflow/docs$
```
Here，
```
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/Split_Learning_for_bank_marketing.po
```